src/main/java/com/google/idea/perf/methodtracer/MethodTracerCommandPredictor.kt

```diff
@@ -68,9 +68,14 @@ class MethodTracerCommandPredictor: CommandPredictor {
     }
 
     private fun predictMethodToken(className: String, token: String): List<String> {
-        val clazz = Class.forName(className)
-        val methodNames = clazz.methods.map { it.name.substringAfter('$') }
-        return predictToken(methodNames, token)
+        return try {
+            val clazz = Class.forName(className)
```

Comment:
This reminds me: `Class.forName()` searches for the class using the classloader that defined the caller (in this case that would be the plugin classloader for ide-perf, since the caller is `MethodTracerCommandPredictor`).

Thus I suspect that method name auto-completion will not work always properly for classes defined in other plugins.

We don't necessarily need to fix this now, but keep it in mind.